config/site_settings.yml

```diff
@@ -529,6 +529,10 @@ users:
     default: false
     client: true
   log_personal_messages_views: false
+  ignore_user_enabled:
+    hidden: true
+    default: false
+    client: true
```

Comment:
Is the site setting used on the client-side?

Reply:
No 👍